gradle/libs.versions.toml

```diff
@@ -12,6 +12,10 @@ androidsvg = "1.4"
 physxjni = "2.6.0"
 physxjswebidl = "2.6.0-1"
 
+# wgpu backend
+wgpu4k = "0.1.0"
+webgpu-ktypes = "0.0.7-SNAPSHOT"
```

Comment:
webgpu-ktypes is a transitive dependency; it will be removed once wgpu4k 0.1.1 is released.